docs/test-ui-bundle.md

```diff
@@ -0,0 +1,23 @@
+# Testing unmerged docs-ui changes
+
+If you want to test an unmerged change made in the `docs-ui` reprository with full content, build as usual the 
```

Comment:
I suggest changing "with full content" to "with content from the ownCloud documentation".